zebra-consensus/src/transaction/check.rs

```diff
@@ -105,3 +108,24 @@ pub fn output_cv_epk_not_small_order(output: &Output) -> Result<(), TransactionE
         Ok(())
     }
 }
+
+/// Check if a transaction is using the diabled sprout pool.
```

Comment:
Incorrect name and docs: Please update the function name and docs to explain which parts of sprout are disabled.

Reply:
Part of https://github.com/ZcashFoundation/zebra/pull/2399/commits/6c1de6d5809a2098101d6ec1c02b0e3ec935afbd